### PR TITLE
Release Notes 6.2.0 (Rolling)

### PR DESCRIPTION
**Pull request by tbsbdr**
- Perma link
- App: External Site
- App: Custom Progress Bar
- Beautified Password Criteria

### CODE REVIEW

**Comment by mmattel** [comment]
@tbsbdr Note that drawio, as far I remember, has been dropped from web as part of the shipment. We mentioned drawio in the release notes in either 6.0 or 6.1. In case, that removal needs mentioning...

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 1]
```diff
@@ -16,26 +16,36 @@
...
+You can now copy links even more easily: As soon as you have shared a file with someone else, you can now just click on copy "Permanent Link". This link allows you to share direct links with people who already have access and enables users to jump straight to the desired file, improving navigation efficiency. Simply copy and share the link to access specific files instantly. (For those who are familiar with the PHP based ownCloud 10: The permanent link in Infinite Scale has the same function as "Private Links" in ownCloud 10)
 
 [discrete]
-=== Issues Fixed
+=== New App: Link to External Sites
 
-* Allow all uploads to restart: Reworks virus handling: https://github.com/owncloud/ocis/pull/9506[#9506]
-* Fix the email notification service: https://github.com/owncloud/ocis/pull/9514[#9514]
+We’ve introduced a new App https://github.com/owncloud/web-extensions/tree/main/packages/web-app-external-sites[External Sites] allowing users to add links to external sites directly from the Apps Menu. This enables easy access to organizational URLs like helpdesk, legal notes, or chat. External sites can be opened in a new tab/window or embedded within ownCloud Web via an iFrame.
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:33]: Please add `Please note that for embedded links you most likely need to adjust your csp.yaml settings.`

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 2]
```diff
@@ -16,26 +16,36 @@
...
 
 [discrete]
-=== Enhancement
+=== New App: Customize your Progress Bar
 
-* Limit concurrent thumbnail requests: https://github.com/owncloud/ocis/pull/9199[#9199]
-* Update web to v8.0.4: https://github.com/owncloud/ocis/pull/9429[#9429]
-* Add cli to purge revisions: https://github.com/owncloud/ocis/pull/9497[#9497]
+Introducing the ability to customize your progress bar at the top of the screen. This extension adds progress bars to the global-progress-bar extension point in ownCloud Web. For demo purposes, we've included a fun Nyan Cat progress bar — enjoy!
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:38]: Please add `After installing the app each user can decide on their own which progress bar should be used. The setting is located on the account page (top right user menu).`

File: modules/ROOT/pages/ocis_release_notes.adoc [hunk 3]
```diff
@@ -232,6 +242,27 @@ The state of the left and right sidebars is now persisted. If you open the left
 * *Custom Permissions* +
 The custom permission selection `View`, `Edit`, `Create`, `Delete`, `Share` in sharing "Custom permissions" will no longer be available.
 
+== Infinite Scale 5.0.6 (Production)
+
+[discrete]
+=== General
+
+This is a patch release only, please update as soon as possible. +
```
Comment (kulmann) [inline modules/ROOT/pages/ocis_release_notes.adoc:250]:
```suggestion
This is a patch release only, please update as soon as possible.
```

Comment (mmattel) [reply modules/ROOT/pages/ocis_release_notes.adoc:250]: Don't commit this,
the `+` symbol is intended to be placed here and part of the asciidoc language.
I can explain details if required 😄

Comment (kulmann) [reply modules/ROOT/pages/ocis_release_notes.adoc:250]: o_O wtf.
no details needed.

**Review by micbar — APPROVED** [review]

**Pull request merged**